src/bindings/pyrpr/src/pyrpr.py

```diff
@@ -197,38 +160,6 @@ def get_first_gpu_id_used(creation_flags):
     raise IndexError("GPU is not used", creation_flags)
 
 
-class array:
```

Comment:
Not used anymore?

Reply:
It was never used. Long time ago I wrote it and planed to use. This is just cleanup.